lib/ContextualPopup/ContextualPopup.js

```diff
@@ -562,7 +578,7 @@ module.exports = kind(
 		// show a transparent scrim for modal popups if
 		// {@link moon.ContextualPopup#scrimWhenModal} is `true`, else show a
 		// regular scrim.
-		if (this.modal && this.scrimWhenModal) {
+		if (this.scrimWhenModal) {
```

Comment:
When the model is true it will return moon.scrimTransparent.make() and create the scrim and when the model turns to false it will is returning  moon.scrim.make(), but ideally to hide the proper scrim it required to return the same moon.scrimTransparent.make().